driver-core/src/test/resources/client-side-encryption/maxWireVersion.json

```diff
@@ -46,6 +46,7 @@
   "tests": [
     {
       "description": "operation fails with maxWireVersion < 8",
+      "skipReason": "waiting on SPEC-1403",
```

Comment:
Removed this accidentally, since it's something that we added in the Java driver's copy, and it got overwritten when we updated the tests